### PR TITLE
fix: list modes

### PR DESCRIPTION
**Pull request by maxwelbm**
- accessing indes of a list directly in some cases causing panic runtime error: index out of range [0] with length 0

### CODE REVIEW

File: pkg/cmd/init/utils.go [hunk 1]
```diff
@@ -78,47 +51,51 @@ func (cmd *InitCmd) selectVulcanTemplates(info *InitInfo) error {
...
 		info.Template = preset
-		info.Mode = strings.ToLower(modes[0])
-		logger.FInfo(cmd.Io.Out, fmt.Sprintf(msg.ModeAutomatic, modes[0], preset))
+		info.Mode = answer
+		return nil
 	}
 
+	var mds string = ""
+	if len(modes) > 0 {
+		logger.Debug("No mode was found for the selected model", zap.Error(err))
```
Comment (PatrickMenoti) [inline pkg/cmd/init/utils.go:91]: "No mode found for the selected preset: " + preset

File: pkg/cmd/init/utils.go [hunk 2]
```diff
@@ -78,47 +51,51 @@ func (cmd *InitCmd) selectVulcanTemplates(info *InitInfo) error {
...
-
 		info.Template = preset
-		info.Mode = strings.ToLower(modes[0])
-		logger.FInfo(cmd.Io.Out, fmt.Sprintf(msg.ModeAutomatic, modes[0], preset))
+		info.Mode = answer
+		return nil
 	}
 
+	var mds string = ""
+	if len(modes) > 0 {
```
Comment (PatrickMenoti) [inline pkg/cmd/init/utils.go:90]: if len(modes) < 1 


You gotta check if there are no modes

File: pkg/cmd/init/utils.go [hunk 3]
```diff
@@ -78,47 +51,51 @@ func (cmd *InitCmd) selectVulcanTemplates(info *InitInfo) error {
...
+		logger.Debug("No mode was found for the selected model", zap.Error(err))
+		return errors.New("No mode was found for the selected template. For more information, run the command again using the '--debug' flag. If the problem persists, contact Azion’s support")
```
Comment (PatrickMenoti) [inline pkg/cmd/init/utils.go:92]: Put this message in a variable in the messages package

**Review by PatrickMenoti — APPROVED** [review]

**Comment by maxwelbm** [comment]
image

**Pull request merged**